src/App.js

```diff
@@ -87,16 +107,21 @@ class App extends React.Component {
   render () {
     return (
       <div className="App scorecard-app">
-        <ScorecardApp onClickCell={this.onClickCell} 
-                      onClickNav={this.onClickNav}
-                      scorecardData={scorecardData}
-                      onClickIcon={this.onClickIcon}
-                      />
-        <InfoModal scorecardData={scorecardData} 
-                   candidate={this.state.candidate}
-                   row={this.state.row}
-                   table={this.state.table}
-                   onClickModalNav={this.onClickModalNav}/>
+        <SunriseNav />
```

Comment:
just in case, could we add a prop like `embed=true` that only renders these when `embed=false` so we have flexibility with whether this is embedded or a full app?

Reply:
Sure, I can add that 👍🏻